diagnostics/ENSO_RWS/LEVEL_04/LEVEL_04.py

```diff
@@ -0,0 +1,74 @@
+#
+#      The code calculates basic Rossby Wave Source terms and associatted variables 
+#          on all available pressure levels 
+#
+#       Contact Information:
+#       PI :  Dr. H. Annamalai,
+#             International Pacific Research Center,
+#             University of Hawaii at Manoa
+#             E-mail: hanna@hawaii.edu
+#
+#       programming :  Jan Hafner,  jhafner@hawaii.edu
+#
+#
+##      This package is distributed under the LGPLv3 license (see LICENSE.txt)
+
+import sys
+import os
+
+import datetime
+
+shared_dir = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+    'shared'
+)
+sys.path.insert(0, shared_dir)
+
+from generate_ncl_call import generate_ncl_call
+
+'''
+      This package is distributed under the LGPLv3 license (see LICENSE.txt)
+
+   ========================================================================
+      input data are as follows:
+     3-dimensional atmospheric variables dimensioned IMAX, JMAX, ZMAX
+     HGT - geopotential height [m]
+     UU  - U  wind  [m/s]
+     VV  - V wind [m/s]
+
+    2-dimensional variables  (fluxes)
+
+      PRECIP precip. kg/m2/sec
+
+    all for full values.
+'''
+
+now = datetime.datetime.now()
+print("=========== LEVEL_04.py =======================================")
+print("   Start of LEVEL_04 calculations  " + now.strftime("%Y-%m-%d %H:%M"))
+print("===============================================================")
+
+##  input data directory
```

Comment:
@jhafner2  You could make a list of file paths and loop through them instead of calling generate_ncl_call several times.